### ISSUE TITLE
Please make it output masks

### ISSUE THREAD

**Issue by worthy7** [BadCafeCode/masquerade-nodes-comfyui#23]
At the moment it outputs an image, maybe a mono-chrome image, because when trying to then convert it to a proper mask with "Image to mask" there is an error saying that 3 tensors (I guess the color channels) are not there. 

![image](https://github.com/BadCafeCode/masquerade-nodes-comfyui/assets/4762399/a5177992-73d3-41ec-bb20-36c93cfbe02d)

**Comment by Matagi1996** [comment]
The whole repo is using "IMAGE" Type for all sorts of Function named Mask***. Not Saying that there is no Reason for this Design, but it was realy confusing me as well.

**Comment by guill** [comment]
Yeah, this code was originally written when ComfyUI was very new and things weren't fully standardized on a `MASK` type. I would definitely use `MASK` if implementing this today, but unfortunately there isn't a way to change that without breaking all existing workflows. As noted in the README, I no longer recommend using this node pack at all unless you specifically have issues installing the more recent (and more idiomatic) packs like Impact Pack.

If you do really have to use this pack, you can convert between images and masks with the included `Image to Mask` node.

**Comment by Matagi1996** [comment]
No Problem, I used the Repo for educational purpose on how to write my own nodes, so thank you very much for letting it stay up even if old.
